src/pug/header.pug

```diff
@@ -1,3 +1,7 @@
+script(src='/insights/static/chrome/js/chrome.js')
+script.
+  window.getName();
```

Comment:
Can you call this function from chrome instead of HTML? I am kinda worried that the chrome file will get loaded twice.